build/zip.py

```diff
@@ -10,6 +10,24 @@
   'libnode.so'
 ]
 
+EXTENSIONS_TO_SKIP = [
+  '.pdb'
+]
+
+PATHS_TO_SKIP = [
+  'angledata',
+  'swiftshader',
+  'resources/inspector'
```

Comment:
i would love to know if these resources are due to a dependency that we shouldn't have. if they are, we should drop the dep rather than excluding them here. if they aren't, we should add a comment here about why we're skipping them.